src/pages/api/matching.ts

```diff
@@ -122,31 +97,32 @@ export default async (req: NextApiRequest, res: NextApiResponse): Promise<void>
         if (!isSessionValid(req, res)) {
             throw new Error('Session is invalid.');
         }
-
         if (!isCookiesUUIDMatch(req, res)) {
             throw new Error('Cookie UUIDs do not match');
         }
-
         if (!req.body.service || !req.body.userfarestages) {
             throw new Error('No service or userfarestages info found');
         }
-
         const service: BasicService = JSON.parse(req.body.service);
         const userFareStages: UserFareStages = JSON.parse(req.body.userfarestages);
-
         const matchingFareZones = getMatchingFareZonesFromForm(req);
-
+        delete req.body.service;
+        delete req.body.userfarestages;
```

Comment:
add a new line here and above so clearer

Reply:
worth adding this comment  
 // Deleting these keys from the object in order to faciliate looping through the fare stage values in the body

Reply:
Added both lines and comment.